src/connecpy/async_client.py

```diff
@@ -97,26 +98,28 @@ async def _make_request(
                     url=self._address + url, content=request_data, **kwargs
                 )
 
-            resp.raise_for_status()
-
             if resp.status_code == 200:
                 response = response_class()
-                response.ParseFromString(resp.content)
+                try:
+                    response.ParseFromString(resp.content)
+                except Exception as e:
+                    raise exceptions.ConnecpyException(
+                        f"Failed to parse response message: {str(e)}"
```

Comment:
This was previously missing for async version

Reply:
oh, thanks!